test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -206,4 +206,19 @@ Describe "Start-Process" -Tags "Feature" {
             $env:TestEnvVariable = $null
         }
     }
+
+    It '-Environment adds or replaces environment variables to child process' {
+        $outputfile = Join-Path -Path $TestDrive -ChildPath output.txt
+        Start-Process pwsh -ArgumentList '-NoProfile','-Nologo','-OutputFormat xml','-Command get-childitem env:' -Wait -Environment @{ a = 1; B = 'hello'; TERM = 'dumb'; PATH = 'mine' } -RedirectStandardOutput $outputfile
```

Comment:
Will $null remove existing env variable?

Reply:
Good question.  Currently you get a null reference exception :)

It makes sense to have `$null` remove the variable, so will make that change and add a test.